### PR TITLE
Don't generate README.html (and others)

### LINKED ISSUE #51: Makefile should not generate README.html
Right now, the Makefile generates README.html from README.md - it shouldn't.

### PR DESCRIPTION
**Pull request by aaren**
Fixes #51.

### CODE REVIEW

**Comment by aaren** [comment]
@gvwilson I thought this might have dropped off your radar :). Can you review?

File: Makefile [hunk 1]
```diff
@@ -22,7 +24,7 @@ motivation.html : motivation.md _layouts/slides.html
 	-o $@ $<
 
 # Pattern to build a generic page.
-%.html : %.md _layouts/page.html
+%.html : %.md _layouts/page.html tools/filters/blockquote2div.py tools/filters/id4glossary.py
```
Comment (gvwilson) [inline Makefile]: Add a variable called FILTERS which is $(wildcard tools/filters/*.py) and them make %.html depend on $(FILTERS) ?

Comment (aaren) [inline Makefile]: done. see latest.

**Comment by jdblischak** [comment]
@aaren, what happened to the code in this PR? The [Makefile](https://github.com/swcarpentry/lesson-template/blob/e07719545d178b3d5007737eb3ca5cf63eac76e6/Makefile) in the core branch of lesson-template no longer excludes the building of README.html.

**Comment by gvwilson** [comment]
Whoops - README.html should be .gitignore'd, and should not be built.  
Please file an issue against lesson-template and I'll see what I broke.

**Comment by zmughal** [comment]
Should `DESIGN.html`, `FAQ.html`, and `LAYOUT.html` also be in `.gitignore`? I see that those also are not in the Git repo, but are still generated.

Edit: Oh, sorry. I didn't see the "(and others)" part of the title.

**Comment by gvwilson** [comment]
If you pull from http://github.com/swcarpentry/lesson-template's "core" 
branch, you'll get updates to Makefile and .gitignore that fix this.
Thanks,
Greg

**Pull request merged**